src/components/CentralContainer/Sidebar/SidebarTabs/SearchTabPanel/index.tsx

```diff
@@ -86,15 +64,20 @@ const SearchTabPanel = () => {
         handleQuerySubmit({queryString: ev.target.value});
     };
 
-    const handleQueryOptionsChange = (
-        _: React.MouseEvent<HTMLElement>,
-        newOptions: QUERY_OPTION[]
-    ) => {
-        setQueryOptions(newOptions);
+    const handleCaseButtonPressed = () => {
```

Comment:
how about `handleCaseSensitivityButtonClick`?
1. `CaseSensitivity` is more accurate than `case`.
2. for `onClick` callbacks, we usually name those as `handle...Click`.